app/lib/get-user-ribbon-data.js

```diff
@@ -1,36 +1,37 @@
 import getAllProjectPreferences from './get-all-project-preferences';
 import getColorFromString from './get-color-from-string';
+import apiClient from 'panoptes-client/lib/api-client'
 
 function getUserRibbonData(user) {
   return getAllProjectPreferences(user).then((prefsResources) => {
     const awaitProjects = Promise.all(prefsResources.map((prefsResource) => {
-      return prefsResource.get('project').catch(() => {
+      return apiClient.type('projects').get(prefsResource.links.project).catch(() => {
```

Comment:
Oh I see, is it because of the added `include` param in the project preferences API call?